flux-se/sylius-payum-stripe-plugin/2.0/config/packages/flux_se_sylius_payum_stripe.yaml

```diff
@@ -0,0 +1,11 @@
+imports:
+    - { resource: "@FluxSESyliusPayumStripePlugin/Resources/config/config.yaml" }
```

Comment:
But you are using a bundle... Why don't you import this file yourself?

Just next to https://github.com/FLUX-SE/SyliusPayumStripePlugin/blob/master/src/DependencyInjection/FluxSESyliusPayumStripeExtension.php#L49

```php
$loader->load('config.yaml');
```

Reply:
@Nyholm I guess you already my comment : https://github.com/symfony/recipes-contrib/pull/1118#issuecomment-817681098

It's to give users of this Sylius plugin a chance to modify for example the `winzou_state_machine` at their convenience, because sometimes  it can not correspond to the processes he needs. `winzou_state_machine` has a big tree and it's very often hard to disable or simply add something to it, that's why most of the time Sylius plugin let this configuration open to modification for the final users. Regarding the `twig`  config it can be removed from the config file and added to a `CompilerPass`.